src/main/java/de/hysky/skyblocker/utils/Utils.java

```diff
@@ -59,7 +59,7 @@ public class Utils {
      * Current Skyblock location (from the Mod API)
      */
     @NotNull
-    private static Location location = Location.UNKNOWN;
+    public static Location location = Location.UNKNOWN;
```

Comment:
Use the getter, making this public could lead to accidental assignment outside of this class.